### PR TITLE
[core] Use get resource instead

### PR DESCRIPTION
**Pull request by yujincheng08**

### CODE REVIEW

File: core/src/main/java/de/robv/android/xposed/XposedInit.java [hunk 1]
```diff
@@ -445,79 +427,17 @@ private static boolean loadNativeLibs(String apk) {
         } catch (IOException e) {
             Log.e(TAG, "  Failed to load native library list from " + apk, e);
             return false;
-        } finally {
-            closeSilently(is);
-            closeSilently(zipFile);
         }
+        closeSilently(is);
```
Comment (aviraxp) [inline core/src/main/java/de/robv/android/xposed/XposedInit.java:431]: Why remove finally block?

Comment (yujincheng08) [reply core/src/main/java/de/robv/android/xposed/XposedInit.java:431]: I think it always be excuted

File: core/src/main/java/de/robv/android/xposed/XposedInit.java [hunk 2]
```diff
@@ -445,79 +427,17 @@ private static boolean loadNativeLibs(String apk) {
         } catch (IOException e) {
             Log.e(TAG, "  Failed to load native library list from " + apk, e);
```
Comment (aviraxp) [inline core/src/main/java/de/robv/android/xposed/XposedInit.java:428]: Not if this is triggered.

Comment (yujincheng08) [reply core/src/main/java/de/robv/android/xposed/XposedInit.java:428]: Oh my fault, it directly returns

**Review by aviraxp — APPROVED** [review]

**Pull request merged**